### PR TITLE
tools: Rename cockpit-shell to cockpit-system

### PR DESCRIPTION
**Pull request by stefwalter**
This is a better catch all name for the components that configure a system. We add a 'Provides' cockpit-shell to ease migration.
    
We also provide a 'Provides' for each subpackage that's bundled.

### CODE REVIEW

**Comment by dperpeet** [comment]
fedora-atomic failure seems related:
```
error: Failed dependencies:
	cockpit-system > 124 is needed by cockpit-ostree-126.x-1.wip.fc24.x86_64
	cockpit-shell >= 122 is needed by cockpit-sosreport-126.x-1.wip.fc24.noarch
	cockpit-shell >= 122 is needed by cockpit-networkmanager-126.x-1.wip.fc24.noarch
	cockpit-shell >= 122 is needed by cockpit-docker-126.x-1.wip.fc24.x86_64
```

**Comment by dperpeet** [comment]
rhel-atomic:
```
error: Failed dependencies:
	cockpit-system > 124 is needed by cockpit-ostree-126.x-1.wip.el7.centos.x86_64
	cockpit-shell >= 122 is needed by cockpit-docker-126.x-1.wip.el7.centos.x86_64
```

**Comment by stefwalter** [comment]
This should fix the dependency issues on the atomics. Doing another test run.

File: tools/cockpit.spec [hunk 1]
```diff
@@ -308,6 +325,78 @@ cat subscriptions.list sosreport.list networkmanager.list selinux.list >> shell.
...
+%endif
+BuildArch: noarch
+
+%description storaged
+The Cockpit component for managing storage.  This package uses Storaged.
+
+%files storaged -f storaged.list
+
+%package system
+Summary: Cockpit Shell user interface package
```
Comment (petervo) [inline tools/cockpit.spec]: Do we want to say system here too?

Comment (stefwalter) [inline tools/cockpit.spec]: Yup good catch.

**Comment by dperpeet** [comment]
Until we build new atomic images and we have implemented the name change downstream, we probably need to add `cockpit-system` to the packages that are force-installed for the atomics: https://github.com/cockpit-project/cockpit/blob/master/test/images/scripts/lib/atomic.install#L39

**Comment by stefwalter** [comment]
> Until we build new atomic images and we have implemented the name change downstream, we probably need to add cockpit-system to the packages that are force-installed for the atomics: https://github.com/cockpit-project/cockpit/blob/master/test/images/scripts/lib/atomic.install#L39

Yup. That's what I did. I have no idea why github reverted back to an old commit. Seems silly. As you can see I just pushed this commit now, and everything is already green. Seems like Semaphore wants to retest though.

**Comment by stefwalter** [comment]
Rebased now so that everything is tested again.

**Comment by stefwalter** [comment]
As far as I know this should not affect OStree composes. Our cockpit-system subpackage contains ```Provides: cockpit-shell```

**Comment by dperpeet** [comment]
Just to be clear, we purposely distance the package `cockpit-system` from the `shell` API component, right? So we leave the documented `shell` intact and as is?

**Comment by stefwalter** [comment]
> So we leave the documented shell intact and as is (guide: https://github.com/cockpit-project/cockpit/blob/master/doc/guide/api-shell.xml#L8)?

Yes. This does not and cannot change. The ```cockpit-bridge --package``` name has not changed.

In addition there's a trello card about allowing an alternate shell/index.html to be installed. Currently that can be configured but not easily driven by package installation:

https://trello.com/c/rs5yvYMC/441-overridable-shell-package

**Pull request closed without merging**